packages/chart/src/components/BarChart.tsx

```diff
@@ -94,6 +97,44 @@ export default function BarChart({ xScale, yScale, seriesScale, xMax, yMax, getX
     return Math.ceil(context.measureText(text).width)
   }
 
+  function cleanData(data) {
```

Comment:
nit pick: cleanData makes sense to me right now, but I'm wondering if we could use a different function name here. Or, maybe we should add comments above the function declaration sharing what this function does in simple terms.